cmd/checkmetrics/ci_slaves/checkmetrics-json-clh.toml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains baseline expectations
+# for checked results by checkmetrics tool.
+#
+# values set specifically for packet.com c1.small slave.
+
+[[metric]]
+name = "boot-times"
+type = "json"
+description = "measure container lifecycle timings"
+# Min and Max values to set a 'range' that
+# the median of the CSV Results data must fall
+# within (inclusive)
+checkvar = ".\"boot-times\".Results | .[] | .\"to-workload\".Result"
+checktype = "mean"
+midval = 1.60
```

Comment:
the midval for qemu is 0.94, @likebreath @sboeuf  are you getting similar numbers?

Reply:
For this we had some idea, even doing manual testing, first thing to check in the todo list if vsock make this difference. But it is nice to have an initial starting data :)

Reply:
For this I am looking to get help of some jeager tracing charts.